proposals/security-mitigation.md

```diff
@@ -0,0 +1,23 @@
+# Proposal: Security Mitigation Standard
+
+## What is a security mitigation standard and what problem it is solving?
+
+* There was no easy way for chart authors to share security mitigation information, chart users not able to read about CVEs present in charts. To fix this broken communication between authors and consumers the security mitigation standard was developed.
+* Proposed [Security Mitigation file](https://github.com/jfrog/chartcenter/blob/master/docs/security-mitigation.yaml) is a way for the chart maintainer to add security mitigation notes for CVEs.
```

Comment:
Let's make sure any files we reference are in the helm org, not an external org.

Reply:
it is a proposal, it cannot be in the helm org

Reply:
I don't understand why a proposal for the helm project has its origins in a vendor's repository.

If this is a standard you wish to propose that the helm community should consider adopting, then it should be contributed to the Helm project (read: the CNCF) to serve as a vendor-neutral location for other parties to collaborate.

If you want to provide attribution to the fact that JFrog proposed the idea, feel free. But I agree with @bridgetkromhout's assessment that any proposals should exist in the helm org.

Reply:
right, can I add the spec and the file to this PR?

Reply:
yes.

Reply:
cool, I will update the PR, thanks for the clarification